docs/_news/news09.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: news
+title: New for Codewind 0.9.0
+description: New for Codewind 0.9.0
+keywords: news, new, updates, update, version, hybrid
+duration: 1 minute
+permalink: news09
+---
+
+## Codewind 0.9.0
+Tuesday, 18 February 2020
+
+#### ✨ New Features and Highlights for 0.9.0 ✨
+Introducing the tech preview of Codewind on the IntelliJ IDE! We want to make our tools available to as many developers as possible in their preferred IDEs. Many people have asked about IntelliJ, as it is a popular option among Java developers, so we are working hard to bring the effortless Codewind experience to IntelliJ. With this tech preview, you can now use Codewind on IntelliJ locally.
+![](images/imagesfornews/intelliJ-techpreview.gif){:width="800"}
+
+**IDEs (Both Eclipse and VS Code)**
+- Detailed application status notifications provide a [link to documentation if a problem is encountered](https://github.com/eclipse/codewind/issues/1812).
+- Depending on the level of support, you can [change the logging level in the IDE](https://github.com/eclipse/codewind/issues/1251). In Eclipse, you can enable support features and set the loggin level in Codewind preferences. In VS Code, you can access this configuration in the command palette - `Set Codewind Server Logging Level`.
```

Comment:
This still implies that you can set the PFE logging level in the Codewind preferences but that is not true.  You can only enable support features in the Codewind preferences.  How about:
In Eclipse, first enable support features in the Codewind preferences and then right click on the connection in the Codewind Explorer view and select `Codewind server log level`.